ansible/inventory/openstack-flex/group_vars/k8s_cluster/k8s-cluster.yml

```diff
@@ -120,6 +120,10 @@ kube_network_node_prefix_ipv6: 120
 kube_apiserver_ip: "{{ kube_service_addresses | ipaddr('net') | ipaddr(1) | ipaddr('address') }}"
 kube_apiserver_port: 6443  # (https)
 
+# The eviction timeout period, defaults to 300 seconds.
+kube_apiserver_pod_eviction_not_ready_timeout_seconds: 60
```

Comment:
this should be a bit longer ... sometimes you might see cases where its reachable but not ready for a bit but working fine etc.

Lets have this at 120 ?

Reply:
yeah, that sounds fair to me

Reply:
do we want both at 120?

Reply:
Updated to 120